share/spice/octopart/content.handlebars

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
For some reason the template doesn't display unless there's a newline or other content before the `if` block -- no idea why